### PR TITLE
Use Maven CI friendly versions

### PR DESCRIPTION
**Pull request by ricardovh**
As described on https://maven.apache.org/maven-ci-friendly.html

This should also fix the problems with Websphere container image dependencies not being resolved (it does locally).

TODO: 
- Check if this does not accidentally break something in CICD/release plugin workings
- Contributing for required maven version

### CODE REVIEW

**Comment by nielsm5** [comment]
You've now turned all parent references into a property defined in the parent. How does the system know what value to resolve the property to if it cannot find the parent?

File: docker/appserver/Jboss-WildFly/pom.xml [hunk 1]
```diff
@@ -16,7 +16,6 @@
 		<dependency>
 			<groupId>org.ibissource</groupId>
 			<artifactId>ibis-adapterframework-ear</artifactId>
-			<version>${iaf.version}</version>
```
Comment (nielsm5) [inline docker/appserver/Jboss-WildFly/pom.xml:19]: <version>${project.version}</version>

**Comment by nielsm5** [comment]
https://medium.com/outbrain-engineering/faster-release-with-maven-ci-friendly-versions-and-a-customised-flatten-plugin-fe53f0fcc0df

**Comment by nielsm5** [comment]
Ik denk dat het zo goed is.

**Review by gvanbrakel — APPROVED** [review]

**Pull request merged**